### PR TITLE
Add HLG encoding format to EXT_gl_colorspace_bt2020

### PR DESCRIPTION
**Pull request by cdglove**
HLG encoding is not included in the original EXT_gl_colorspace_bt2020 extension. This change adds a new token to specific the format, and adds language specifying how applications can use the format.

I used a free token just above the linear format so that they're all next to each other. I then ordered the spec language by token order, which results in the HLG format being specified first.

I decided to add this to the existing EXT rather than make a whole new EXT, but I'm happy to make a new EXT
if that's preferable.

### CODE REVIEW

**Comment by stonesthrow** [comment]
Add some original authors for review.

File: api/egl.xml
Comment (stonesthrow) [inline api/egl.xml:835]: Since this enum is reserved for James, he'll have to give an OK.

File: extensions/EXT/EGL_EXT_gl_colorspace_bt2020_linear.txt
Comment (stonesthrow) [inline extensions/EXT/EGL_EXT_gl_colorspace_bt2020_linear.txt:96]: Does expanding the acronym add any value?:
"HLG (Hybrid Log Gamma) ..."

Comment (cdglove) [reply extensions/EXT/EGL_EXT_gl_colorspace_bt2020_linear.txt]: Done.

**Review by stonesthrow — APPROVED** [review]

**Comment by stonesthrow** [comment]
Can we get a +1 from reviewers?

**Comment by cdglove** [comment]
Ping.

The changes seem to be generally unopposed, can I can some +1s from others? Or some feedback of changes are necessary?

**Comment by janharaldfredriksen-arm** [comment]
Was the question on enum reservation above answered? Otherwise it seems straightforward enough to me.

**Comment by stonesthrow** [comment]
@cubanismo , James give us a +1 for using one of your enums. This change is simple enough, ready to approve. (Note James may be in Osaka for F2F)

**Comment by stonesthrow** [comment]
Ping for a review

File: api/egl.xml
Comment (cubanismo) [reply api/egl.xml:835]: Sorry, this enum is already in use by a somewhat provisional extension that we've been shipping for a while:

https://github.com/aritger/eglstreams-kms-example/blob/master/proposed-extensions/EGL_NV_output_drm_flip_event.txt

**Review by cubanismo — CHANGES_REQUESTED** [review]
Needs different enum value.

File: api/egl.xml
Comment (stonesthrow) [reply api/egl.xml:835]: There are a few unused in this block reserved for James, which does have colorspace related enums, if OK with James; or you will have to request a new block of enums from Jon. Just make a PR with egl.xml, and block at end - looks like x3540- is start of free enums

Comment (cdglove) [reply api/egl.xml:835]: Ack. Thank you James. It means they're not continuous, which is unfortunate, but fine. I'll grab a new one and update the PR.

File: api/egl.xml
Comment (stonesthrow) [inline api/egl.xml:835]: Undo this change

**Review by stonesthrow — CHANGES_REQUESTED** [review]
Needs rebase and updates

**Review by stonesthrow — APPROVED** [review]

File: api/egl.xml
Comment (cdglove) [reply api/egl.xml]: Fixed.

File: extensions/EXT/EGL_EXT_gl_colorspace_bt2020_linear.txt
Comment (cdglove) [reply extensions/EXT/EGL_EXT_gl_colorspace_bt2020_linear.txt]: Done.

**Comment by cdglove** [comment]
Sorry, I pushed a bad merge. I've corrected everything now, thanks for the review Jeff.

**Review by stonesthrow — APPROVED** [review]

**Comment by cdglove** [comment]
Ping.

And possibly some help -- I don't know how to resolve the "Changes requested" box, in case that's what's holding things up.

**Comment by stonesthrow** [comment]
Hmm, I thought they went away on next change/upload. The icon does not show next to James' name on the upper right reviewr's area. Which it will. So seems odd.

**Comment by stonesthrow** [comment]
Sorry. James. Just pushing buttons to see what they do.

**Comment by stonesthrow** [comment]
I think you or James just needed to resolve the comment/review to close the action item.

**Comment by stonesthrow** [comment]
Lets get one more approve, and then we can set "Approved to Merge" and assign to Jon to merge.

**Review by stonesthrow — APPROVED** [review]

**Comment by cdglove** [comment]
Thank you! Seems to be fixed now. Strange glitch.

**Comment by stonesthrow** [comment]
I pulled the trigger to move forward.

File: index.php
Comment (oddhack) [inline index.php:274]: This is generated from registry.tcl, so after merging I'll tweak that to be functionally equivalent.

**Comment by cdglove** [comment]
Thank you both!

**Pull request merged**
